### PR TITLE
Piece: Postgres ETL

### PR DESCRIPTION
**Pull request by prrao87**
This PR adds a new project to insert data into a Postgres table and measure the performance of async querying. It would be interesting to see the async data ingestion and query performance using `asyncpg` in Python and `sqlx` in Rust.

### CODE REVIEW

File: python/postgres_etl/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+line-length = 100
+indent-width = 4
+
+[tool.ruff.lint]
+select = ["I"]
+ignore = ["E501"]
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
```
Comment (sanders41) [inline python/postgres_etl/pyproject.toml:13]: If you add this in the `pyproject.toml` then you won't need to add the `@pytest.mark.asyncio` decorator to tests.

```toml
[tool.pytest.ini_options]
asyncio_mode = "auto"
```

File: python/postgres_etl/test_main.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+from load_data import run as loader
+from main import perf_query, summary_query
+
+
+load_dotenv()
+PG_PASSWORD = os.environ.get("POSTGRES_PASSWORD")
+PG_URI = f"postgres://postgres:{PG_PASSWORD}@localhost:5432/etl"
+
+
+@pytest.mark.asyncio
```
Comment (sanders41) [inline python/postgres_etl/test_main.py:18]:
```suggestion
```

File: python/postgres_etl/test_main.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+async def test_main():
+    counter = await loader()
+    assert counter == 1000
+
+
+@pytest.mark.asyncio
```
Comment (sanders41) [inline python/postgres_etl/test_main.py:24]:
```suggestion
````

File: python/postgres_etl/test_main.py [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+async def test_summary_query():
+    conn = await asyncpg.connect(PG_URI)
+    count = await summary_query(conn)
+    assert count == 1000
+
+
+@pytest.mark.asyncio
```
Comment (sanders41) [inline python/postgres_etl/test_main.py:31]:
```suggestion
```

**Review by sanders41 — COMMENTED** [review]
I haven't gone though all of this PR, but was glancing through out of curiosity. This is just a small suggestion to make your python async testing easier.

**Comment by prrao87** [comment]
@sanders41 that's a good tip, thanks!

**Pull request closed without merging**